README.rst

```diff
@@ -6,23 +6,19 @@
 
 |
 
-**QAT** (Quantum Assembly Toolchain) is a low-level quantum compiler and runtime which facilitates executing quantum IRs
+**QAT** (Quantum Assembly Toolkit) is a low-level quantum compiler and runtime which facilitates executing quantum IRs
```

Comment:
How come we're renaming the package?

Reply:
We're not, it's still QAT in all meaningful places, its acronym was always toolkit/toolchain. I'm just locking in a particular name since it fits better.

Reply:
I know some of the socials used toolkit but the blog post and central comms used toolchain as that was the "original" definition. Happy for this to be changed but think it should be a shared conscious decision and updated across the docs/comms etc.

Reply:
I'll just add both so people can use whatever. It's switched a few times for me during development - are we a tool chain or are we a collection of tools that are used distinctly - but I think latetly we're definitely leaning towards toolkit, not chain.